cli/command/system/info.go

```diff
@@ -135,9 +135,11 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 		fmt.Fprintf(dockerCli.Out(), "Default Runtime: %s\n", info.DefaultRuntime)
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "Security Options:")
-	ioutils.FprintfIfNotEmpty(dockerCli.Out(), " %s", strings.Join(info.SecurityOptions, " "))
-	fmt.Fprintf(dockerCli.Out(), "\n")
+	if info.OSType != "windows" {
+		fmt.Fprintf(dockerCli.Out(), "Security Options:")
+		ioutils.FprintfIfNotEmpty(dockerCli.Out(), " %s", strings.Join(info.SecurityOptions, " "))
+		fmt.Fprintf(dockerCli.Out(), "\n")
```

Comment:
If we would decide to make this not *platform* specific, but to only print `Security Options:` if there's actually any (looks like we do that for "runtimes" above, so we're not really consistent), this while block could simply be;

```go
ioutils.FprintfIfNotEmpty(dockerCli.Out(), "Security Options: %s\n", strings.Join(info.SecurityOptions, " "))
```